docs/datatypes/video/index.md

```diff
@@ -81,6 +81,22 @@ for idx, c in enumerate(d.tensor):
 
 Makes sense, right?
```

Comment:
```suggestion
```

Not related to this PR, but there are always some people to whom things don't make sense, let's not mock them lol